AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -94,10 +94,11 @@ - (void)applyEnabledState:(BOOL)isEnabled {
       eTag = [MS_USER_DEFAULTS objectForKey:kMSIdentityETagKey];
     }
     NSString *authToken = [self retrieveAuthToken];
+    NSString *accountId = [self retrieveAccountId];
 
-    // Only set the auth token if it is not nil to avoid triggering callbacks.
-    if (authToken) {
-      [MSAuthTokenContext sharedInstance].authToken = authToken;
+    // Only set the auth token if it and account id are not nil to avoid triggering callbacks.
```

Comment:
Since you are changing the comment,
`if it and account id are` -> `if auth token and account id are`